workspaces/arborist/README.md

```diff
@@ -333,3 +333,13 @@ pruning nodes from the tree.
 
 Note: `devOptional` is only set in the shrinkwrap/package-lock file if
 _neither_ `dev` nor `optional` are set, as it would be redundant.
+
+## BIN
+
+Arborist ships with a cli that can be used to run arborist specific commands outside of the context of the npm CLI. This script is currently not part of the public API and is subject to breaking changes outside of major version bumps.
```

Comment:
why advertise it if it's not part of the public API, slash, why not just let it be part of it?

Reply:
I didn't know about the cli here until it was explicitly pointed out to me in the past, and it has proved helpful for a lot of debugging and exploration work I've done. So I felt there was value in at least pointing it out, and a command to show the most up to date usage info. 

But since it is currently has 0% test coverage, I don't feel confident in making it a public API at this point.

Reply:
If the plan is to add such coverage and make it part of the API, then, great! :-D